### ISSUE TITLE
Realtime view shows error

### ISSUE THREAD

**Issue by holtwick** [mikecao/umami#788]
Under `/realtime` I currently see *An unexpected error has occurred.*

It started showing up after I did send a wrong URL to the wrong project using `/collect` API using https://github.com/mikecao/umami/issues/787 I probably have also missed out some properties.

In the console I see the following output:

<img width="1278" alt="20210916-111604-capture-holtwick@2x" src="https://user-images.githubusercontent.com/49557/133585790-55b8f4b4-e708-445b-b1ad-5e5241cc210b.png">

I use a Docker container and didn't find a way on it to get more log files.

**Comment by mikecao** [comment]
Will need more information. The error is complaining about some localization messages. Not sure what would cause that.

**Comment by holtwick** [comment]
I tried to narrow it down a little bit:

- I succeed getting into realtime when resetting all local data (localStorage etc.) and logging in again
- Then it only shows the error on the tab "Visitors"
- It only shows the error for one website

This website is the only one using capturing events. I use the PHP Script I described in a previous ticket. These are the events collected:

<img width="417" alt="20211004-140822-capture-holtwick@2x" src="https://user-images.githubusercontent.com/49557/135848776-a7a4ffdc-1af9-4dc5-b732-3a5b60c8656c.png">

Hope that helps.
